sql/log_event_client.cc

```diff
@@ -1111,8 +1131,9 @@ void Rows_log_event::change_to_flashback_event(PRINT_EVENT_INFO *print_event_inf
   Table_map_log_event *map;
   table_def *td;
   DYNAMIC_ARRAY rows_arr;
-  uchar *swap_buff1;
   uchar *rows_pos= rows_buff + m_rows_before_size;
+  Field_info *ai_fields = nullptr;
+  Field_info *bi_fields = nullptr;
```

Comment:
Likewise with the above two, no space before the equals

Reply:
fixed